src/assets/scss/maps/_color-map.scss

```diff
@@ -36,4 +36,5 @@ $color-map: (
   'vocab-background': rgba(220, 220, 220, 0.68),
   'warning': #f0ad4e,
   'white': #ffffff,
+  'how-to-multi-form': #000
```

Comment:
The values here are ordered alphabetically based on their key, so please put it in the right place.